zkevm-circuits/src/state_circuit/state.rs

```diff
@@ -1167,6 +1167,37 @@ pub struct StateCircuit<
     storage_ops: Vec<Operation<StorageOp>>,
 }
 
+impl<
+        const GLOBAL_COUNTER_MAX: usize,
+        const MEMORY_ROWS_MAX: usize,
+        const MEMORY_ADDRESS_MAX: usize,
+        const STACK_ROWS_MAX: usize,
+        const STACK_ADDRESS_MAX: usize,
+        const STORAGE_ROWS_MAX: usize,
+    >
+    StateCircuit<
+        GLOBAL_COUNTER_MAX,
+        MEMORY_ROWS_MAX,
+        MEMORY_ADDRESS_MAX,
+        STACK_ROWS_MAX,
+        STACK_ADDRESS_MAX,
+        STORAGE_ROWS_MAX,
+    >
+{
+    #[allow(missing_docs)]
+    pub fn new(
```

Comment:
Could you just add some doc lines mentioning that the fn generates a new `StateCircuit` instance containing the operations passed?

On that way we don't need the `#[allow(...)]`

Reply:
SURE